docker-compose.yml

```diff
@@ -44,7 +44,6 @@ services:
       CELERY_BROKER_URL: redis://redis:6379/0
       EVENTBRITE_APP_KEY: ${EVENTBRITE_APP_KEY}
       DATABASE_URL: postgres://beluga:beluga@postgres:5432/beluga
-      COLLECTION_INTERVAL: 20
```

Comment:
Could we add a comment with units here (or some other appropriate place, maybe a README)?